README.md

```diff
@@ -61,6 +61,7 @@ Name | Description
 `failed_manifest_prefix` | A file prefix that will be used for manifest logs on failure, defaults to `failed`.
 `failure_topic_name` | An SNS topic name that will be notified about batch processing failures, defaults to `ControlshiftLambdaLoaderFailure`.
 `failure_topic_name_for_run_glue_job_lambda` | An SNS topic name that will be notified about batch processing failures, defaults to `ControlshiftLambdaLoaderFailure`.
+`glue_physical_connection_requirements` | An object as defined in terraform [aws_glue_connection's physical_connection_requirements](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/glue_connection#physical_connection_requirements) property for setting up Glue jobs' connection to Redshift.
```

Comment:
Do we have any guidance on how users should determine these values? Are the security group and subnet something we can potentially create or infer from other resources?

Reply:
I think we can get the `availability_zone` and `security_group_id_list` values from `aws_redshift_cluster.default`.
For the `subnet_id` it's trickier because the `aws_redshift_cluster` resource only exposes a [`cluster_subnet_group_name`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/redshift_cluster#cluster_subnet_group_name) and there is data resource we can use to extract the associated subnets. Strangely, terraform docs claim the following with regards to cluster_subnet_group_name:

> If this parameter is not provided the resulting cluster will be deployed outside virtual private cloud (VPC).

But that's not really the case, at least for the cluster we created recently, it's within the VPC and associated with 5 subnets.